pwa-shell/src/state/content/selectors.js

```diff
@@ -30,6 +30,21 @@ export const getSelectedContentId = createSelector(
   content => get(content, 'selected.id')
 );
 
+export const getSelectedContentIdxFromList = createSelector(
+  [getSelectedContentId, getContentList],
+  (contentId, contentList) => contentList.findIndex(el => el.id === contentId),
+);
+
+export const getNextToSelectedContent = createSelector(
+  [getSelectedContentIdxFromList, getContentList],
```

Comment:
if "idx" means "index", please use index so it cannot be confused with "id" that is also used here